### PR TITLE
CHAT-591 Fix avatars dimensions in Chat right panel

### PR DESCRIPTION
**Pull request by sayachi**

### CODE REVIEW

File: application/src/main/java/org/exoplatform/chat/portlet/notification/assets/chat-mini.less [hunk 1]
```diff
@@ -284,7 +284,6 @@
     .border-radius(@widthOfAvatar);
   }
   .msAvatarLink, .msAvatarLink > img {
-    .square(@widthOfAvatar);
   }
```
Comment (thomasdelhomenie) [inline application/src/main/java/org/exoplatform/chat/portlet/notification/assets/chat-mini.less]: You can remove the block since it is empty now.

File: application/src/main/webapp/js/chat.js [hunk 2]
```diff
@@ -3653,7 +3653,7 @@ ChatApplication.prototype.renderRoomUser = function(user, showOfflineUsers) {
     }
     html += ">";
     html += "  <div class='msUserAvatar pull-left'>";
-    html += "    <span class='msAvatarLink avatarCircle'><img onerror=\"this.src='/chat/img/user-default.jpg'\" src='/" + eXo.env.portal.rest +"/v1/social/users/" + user.name + "/avatar' alt='" + user.fullname + "'></span>";
+    html += "    <span class='msAvatarLink avatarCircle'><img onerror=\"this.src='/chat/img/user-default.jpg'\" src='/" + eXo.env.portal.rest +"/v1/social/users/" + user.name + "/avatar' alt='" + user.fullname + "'></span>";    html += "  </div>";
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: Is this change needed ? It seems to break the DOM.

File: application/src/main/webapp/js/chat-modules.js [hunk 3]
```diff
@@ -547,9 +547,9 @@ ChatRoom.prototype.showMessages = function() {
             if (prevUser !== "") {
               if (prevUser !== "__system") {
                 if (thiss.isPublic) {
-                  out += "  <a class='msAvatarLink avatarCircle' href='#'><img src='/chat/img/support-avatar.png'></a>";
+                    out += "  <a class='msAvatarLink avatarCircle' href='#'><img src='/chat/img/support-avatar.png'></a>";
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat-modules.js]: All the changes of this file are indentation. Not needed in this PR.

File: application/src/main/webapp/js/chat.js (hunk 2 above)
Comment (sayachi) [inline application/src/main/webapp/js/chat.js]: PR updated

**Comment by sayachi** [comment]
Merged in develop
